contribs/github-bot/config.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"github.com/gnolang/gno/contribs/github-bot/internal/client"
+	c "github.com/gnolang/gno/contribs/github-bot/internal/conditions"
+	r "github.com/gnolang/gno/contribs/github-bot/internal/requirements"
+)
+
+// Automatic check that will be performed by the bot
+type automaticCheck struct {
+	description string
+	ifC         c.Condition   // If the condition is met, the rule is displayed and the requirement is executed
```

Comment:
I think you used `ifC` because otherwise `if` is a keyword. What if we just made this exported? `If` / `Then`?

Reply:
Yes exactly and it was exported before.
What do you think @zivkovicmilos? Are you for exporting them back or something else?

Reply:
I mean, the naming is honestly fine 🤷‍♂️ 
I don't see why we would export a field just to not catch a keyword conflict